packages/dapp/src/components/Widget/Widget.tsx

```diff
@@ -3,5 +3,7 @@ import { useWidgetConfig } from '../../hooks/';
 
 export function Widget({ starterVariant }) {
   const widgetConfig = useWidgetConfig((starterVariant = { starterVariant }));
-  return <LiFiWidget config={widgetConfig} />;
+  return (
+    <LiFiWidget config={widgetConfig} integrator={'testnets-jumper.exchange'} />
```

Comment:
We should provide an integrator string based on the MODE variable to have the same code in every branch (testnet/dev/staging/main) without limitations.